recipes/model-index/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "model-index" %}
+{% set version = "0.1.11" %}
+{% set commit = "09c7865ab2cbe00827d3af879a2faf7d286cb1b7" %}
```

Comment:
Can you ask upstream for a stable tag/release?

Reply:
I have opened [two issues](https://github.com/paperswithcode/model-index/issues) on the repo since last week, but the maintainer is not responsive. The repo have no update since June 2021, but the package is being used by 221 repos. The package [sdist](https://github.com/paperswithcode/model-index/issues) on PyPI is missing several files (LICENSE and requirements.txt), and therefore it can't be used for building the recipe. I use the commit id as a temporary workaround. If the maintainer makes a stable/tag release, I can change the recipe to use that. We need this package on conda-forge so that downstream recipes (openmim, autogluon.multimodal, autoglon) can get on conda-forge as well. 


I have also asked the openmim package maintainer to make model-index as an optional dependency, but it is not option. We are stuck in this model-index package. 
https://github.com/open-mmlab/mim/issues/193

Reply:
> The package [sdist](https://github.com/paperswithcode/model-index/issues) on PyPI is missing several files (LICENSE and requirements.txt), and therefore it can't be used for building the recipe.

B/c that one is "frozen" and reproducible we should add the files that are missing instead of jumping into a specific commit that may or may not correspond to the published version. You can source those files directly from GH. See https://github.com/conda-forge/types-jsonschema-feedstock/blob/main/recipe/meta.yaml#L10 for an example.

Reply:
@ocefpaf Thank you for the advice. I have made changes as suggested. The recipe has passes linux and osx builds. It is ready to be reviewed and merged. Thank you.